bots/image-refresh

```diff
@@ -28,79 +28,79 @@ from task import github, REDHAT_STORE
 
 TRIGGERS = {
     "centos-7": [
-        "cockpit/centos-7@cockpit-project/starter-kit",
+        "centos-7@cockpit-project/starter-kit",
     ],
     "continuous-atomic": [
-        "cockpit/continuous-atomic@cockpit-project/cockpit-ostree",
+        "continuous-atomic@cockpit-project/cockpit-ostree",
     ],
     "debian-testing": [
-        "verify/debian-testing"
+        "debian-testing"
     ],
     "debian-stable": [
-        "verify/debian-stable"
+        "debian-stable"
     ],
     "fedora-29": [
-        "verify/fedora-atomic",
-        "cockpit/fedora-29@cockpit-project/cockpit-podman",
+        "fedora-atomic",
+        "fedora-29@cockpit-project/cockpit-podman",
     ],
     "fedora-30": [
-        "verify/fedora-30",
-        "selenium/chrome",
-        "selenium/firefox",
-        "selenium/edge",
-        "avocado/fedora",
-        "container/bastion",
-        "cockpit/fedora-30@cockpit-project/starter-kit",
-        "cockpit/fedora-30@cockpit-project/cockpit-podman",
-        "cockpit/fedora-30@weldr/lorax",
-        "cockpit/fedora-30/live-iso@weldr/lorax",
-        "cockpit/fedora-30/qcow2@weldr/lorax",
-        "cockpit/fedora-30/chrome@weldr/cockpit-composer",
-        "cockpit/fedora-30/firefox@weldr/cockpit-composer",
-        "cockpit/fedora-30/edge@weldr/cockpit-composer",
+        "fedora-30",
+        "fedora-30/selenium-chrome",
+        "fedora-30/selenium-firefox",
+        "fedora-30/selenium-edge",
+        "fedora-30/avocado-fedora",
```

Comment:
This should be just `fedora-30/avocado-fedora`, right? the `-fedora` is pointless there.

Reply:
Right, let's make this just `/avocado` (I suppose that's what you actually meant).